azure/Guidelines.md

```diff
@@ -760,79 +760,64 @@ while the Microsoft guidelines use the name "Operation-Location".
+A common pattern when working with multiple files is for your service to receive requests that contain the location(s) of files to process ("input") and a location(s) to place any files that result from processing ("output"). Note: the terms "input" and "output" are just examples; use terms more appropriate to your service's domain.
 
-A common pattern when working with multiple files is for your service to receive requests that contain the location(s) of files to process, e.g. "input" and a location(s) to place the any files that result from processing, e.g. "output." (Note: the terms "input" and "output" are just examples and terms more relevant to the service domain are more appropriate.)
-
-For example, in a request payload may look similar to the following:
+For example, a service's request body to configure BYOS may look like this:
 
 ```json
 {
-"input":{
+  "input":{
     "location": "https://mycompany.blob.core.windows.net/documents/english/?<sas token>",
+    "delimiter": "/",
+    "lastModified": "Wed, 21 Oct 2015 07:28:00 GMT"
+  },
+  "output":{
+    "location": "https://mycompany.blob.core.windows.net/documents/spanish/?<sas token>",
     "delimiter":"/"
-    },
-"output":{
-    "location": "https://mycompany.blob.core.windows.net/documents/spanglish/?<sas token>",
-    "delimiter":"/"
-    }
+  }
 }
 ```
-Note: How the service gets the request body is outside the purview of these guidelines.
 
-Depending on the requirements of the service, there can be any number of "input" and "output" sections, including none. However, for each of the "input" sections the following apply:
+Depending on the requirements of the service, there can be any number of "input" and "output" sections, including none. 
 
-:white_check_mark: **DO** include a JSON object that has string values for "location" and "delimiter."
+:white_check_mark: **DO** include a JSON object that has string values for "location" and "delimiter". For "location", the customer must pass a URL to a blob prefix which represents a directory. For "delimiter", the customer must specify the delimiter character they desire to use in the location URL; typically "/" or "\". 
 
-:white_check_mark: **DO** use a URL to a blob prefix with a container scoped SAS on the end with a minimum of `listing` and `read` permissions.
+:heavy_check_mark: **YOU MAY** support the "lastModified" field for input directories (see guideline above).
 
-For each of the "output" sections the following apply:
+:white_check_mark: **DO** support a "location" URL with a container-scoped SAS that has a minimum of `listing` and `read` permissions for input directories.
 
-:white_check_mark: **DO** use a URL to a blob prefix with a container scoped SAS on the end with a minimum of `write` permissions
+:white_check_mark: **DO** support a "location" URL with a container-scoped SAS that has a minimum of `write` permissions for output directories.
```

Comment:
What are your thoughts on [`write` vs. `create`](https://docs.microsoft.com/en-us/rest/api/storageservices/create-service-sas#permissions-for-a-directory-container-or-blob) here?  Is `create` a little safer if you don't want to trash anything already in `output.location`?  Or should we recommend `write` because it'll always get the job done?

Reply:
We agreed to leave this as is